x/rollapp/keeper/rollapp.go

```diff
@@ -62,6 +62,8 @@ func (k Keeper) CheckAndUpdateRollappFields(ctx sdk.Context, update *types.MsgUp
 
 		if update.GenesisInfo.GenesisAccounts != nil {
 			current.GenesisInfo.GenesisAccounts = update.GenesisInfo.GenesisAccounts
+		} else if current.GenesisInfo.GenesisAccounts != nil {
```

Comment:
```suggestion
		} else  {
```

Reply:
If GenesisAccounts is nil, I can get a panic, right?